From ebe62b178f3e5fcde8a311e64aaffe62099204a5 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Sun, 21 Apr 2019 12:44:26 +0200
Subject: [PATCH] make -Werror into an optional flag

This lets distros disable the flag as random errors might come up with
different compiler flags and older/newer toolchain versions.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 harness/Makefile | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/harness/Makefile b/harness/Makefile
index f477737..a155c4b 100644
--- a/harness/Makefile
+++ b/harness/Makefile
@@ -6,7 +6,8 @@ PROGS:=$(PARTPROGS) $(EXTRAPROGS)
 HARNESS_SRCS:=main.c
 # io_queue.c
 
-CFLAGS+=-Wall -Werror -I../src -g -O2 -DPAGE_SIZE=$(shell getconf PAGESIZE)
+CFLAGS_WERROR?=-Werror
+CFLAGS+=-Wall $(CFLAGS_WERROR) -I../src -g -O2 -DPAGE_SIZE=$(shell getconf PAGESIZE)
 #-lpthread -lrt
 
 all: $(PROGS)
-- 
2.21.0

